packages/plugins/connection-manager/screens/settings.ts

```diff
@@ -28,6 +28,11 @@ export default class SettingsWebview extends WebviewProvider {
 
   private updateConnection = async ({ connInfo, isGlobal, editId }) => {
     console.log({ connInfo, isGlobal, editId });
+
+    if (connInfo.database) {
```

Comment:
We need to check if connInfo.dialect is SQLite.

Adding this change, we are good to go! Thanks @mitchellsimoens !